src/components/MarkdownProvider/components/PropSheets.tsx

```diff
@@ -44,37 +57,19 @@ export const PropSheets: React.FC<{ data: ComponentDoc[]; reactPackage: IPackage
 
                   return (
                     <Row key={`${propSheet.displayName}-${propSheetKey}`}>
-                      <Cell
-                        css={css`
-                          color: ${p => getColor('kale', 400, p.theme)};
-                          font-family: ${p => p.theme.fonts.mono};
-                        `}
-                      >
-                        {prop.name}
-                      </Cell>
-                      <Cell
-                        css={css`
-                          color: ${p => getColor('red', 600, p.theme)};
-                          font-family: ${p => p.theme.fonts.mono};
-                        `}
-                      >
-                        {prop.type.name}
-                      </Cell>
-                      <Cell
-                        css={css`
-                          font-family: ${p => p.theme.fonts.mono};
-                        `}
-                      >
-                        {prop.defaultValue ? prop.defaultValue.value : '-'}
+                      <Cell>
+                        <Code>{prop.name}</Code>
                       </Cell>
                       <Cell>{prop.description}</Cell>
```

Comment:
Seems odd to have the description rendered with monospace font. Once Garden has a `<Span isMonospace>` component (next week?) this will be a good place to apply rather than overriding the table `<Row>` with blanket `font-family` styling. In the mean time, I'd recommend surrounding cell content with `<MD isMonospace>` just for the columns that need it.